raiden/network/proxies/token_network.py

```diff
@@ -2158,6 +2176,9 @@ def settle(
         partner_locksroot: Locksroot,
         given_block_identifier: BlockSpecification,
     ):
+
+        channel_identifier = channel_settle_event.channel_identifier
```

Comment:
why the settle function must be modified? this is related with the previous comment